### DISCUSSION TITLE
Checkout Block doesn't support PayPal Express

### DISCUSSION THREAD

**Discussion by itielsefer14** [woocommerce/woocommerce#45070] [WooCommerce, Blocks, and Site Editing]
## hola! el error es que no acepta otras formas de pago, como por ejemplo PayPal express etc

**Comment by senadir** [comment]
Hi @itielsefer14 ,

The Cart and Checkout blocks are still in preview and there are a number of extensions and payment methods that are not integrated yet. Paypal Express is one of those extensions. You can follow [this document](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/) to keep up to date with the latest status of the blocks.

I'm going to close this issue as there's nothing actionable for it at the moment. We are aware of there being limited payment method support and it's on our roadmap to integrate WooCommerce owned extensions as well as encourage third party developers to integrate their own extensions.

Thanks for taking the time to create the issue and make sure we were aware!

**Comment by ellenbauer** [comment]
I think support for Paypal is really key, it's still hasn't been added to the supported plugins for checkout.

**Comment by ralucaStan** [comment]
According to this https://github.com/woocommerce/woocommerce-paypal-payments/issues/211#issuecomment-1551145433 a new pre-release for the next PayPal Payments 2.0.5 is available [on GitHub](https://github.com/woocommerce/woocommerce-paypal-payments/releases/tag/2.0.5-rc1) with express payment methods support in the WooCommerce blocks.
